docs/content/scripts/content/google-maps.md

```diff
@@ -34,8 +34,8 @@ You'll need an API key with permissions to access the [Static Maps API](https://
 
 Showing an interactive JS map requires the Maps JavaScript API, which is a paid service. If a user interacts with the map, the following costs will be incurred:
 - $7 per 1000 loads for the Maps JavaScript API (default for using Google Maps)
-- $5 per 1000 loads for the Geocoding API
-- $2 per 1000 loads for the Static Maps API
+- $2 per 1000 loads for the Static Maps API - You can avoid providing a `placeholder` slot.
+- $5 per 1000 loads for the Geocoding API - You can avoid this by providing a `google.maps.LatLng` object instead of a string for the `center` prop
```

Comment:
nit: can we link to the source, where the amounts are coming from?
This might get outdated as time goes by, maybe we could add a disclaimer along the lines: "The example cost is based on this source. Actual cost may change. For more accurate cost estimation, please visit ... "